styleguide_example/files/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.conf import settings
+
+from styleguide_example.common.models import BaseModel
+
+from styleguide_example.users.models import BaseUser
+
+from styleguide_example.files.utils import file_generate_upload_path
+
+
+class File(BaseModel):
+    file = models.FileField(upload_to=file_generate_upload_path)
+    file_name = models.CharField(max_length=255)
+    file_type = models.CharField(max_length=255)
+
+    uploaded_by = models.ForeignKey(BaseUser, on_delete=models.CASCADE)
```

Comment:
uploaded_at? The verify step will add an uploaded at. Via uploaded_at we will distinguish the successfully uploaded files